### PR TITLE
Workflows to deploy and remove custom env from branches 

### PR DESCRIPTION
**Pull request by azatsafin**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [x] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [ ] My changes generate no new warnings(e.g. Sonar is happy)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: .github/workflows/branch-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: DeployFromBranch
+on:
+  workflow_dispatch:
+  pull_request:
+    types: ['labeled']
+jobs:
+  build:
+    if: ${{ github.event.label.name == 'status/feature_testing' }}
```
Comment (Haarolean) [inline .github/workflows/branch-deploy.yml:8]: I've created a label btw

File: .github/workflows/branch-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          cache-to: type=local,dest=/tmp/.buildx-cache
+    outputs:
+      tag: ${{ steps.extract_branch.outputs.tag }}
+  make-branch-env:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: clone
+        run: |
+          git clone https://azsafin:${{ secrets.KAFKA_UI_INFRA_TOKEN }}@gitlab.provectus.com/provectus-internals/kafka-ui-infra.git
```
Comment (Haarolean) [inline .github/workflows/branch-deploy.yml:81]: so what about a separate "technical" account for this?

File: .github/workflows/branch-remove.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+name: DeployFromBranch
```
Comment (Haarolean) [inline .github/workflows/branch-remove.yml:1]: Undeploy?

**Review by Haarolean — CHANGES_REQUESTED** [review]

File: .github/workflows/branch-deploy.yml (hunk 2 above)
Comment (azatsafin) [reply .github/workflows/branch-deploy.yml:81]: let do it as separate step

**Pull request merged**